src/Evolution/Systems/ScalarAdvection/System.hpp

```diff
@@ -0,0 +1,20 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+/*!
+ * \ingroup EvolutionSystemsGroup
+ * \brief Items related to evolving the scalar advection equation.
+ *
+ * \f{align*}
+ * \partial_t U + \nabla \cdot (v U) = 0
+ * \f}
+ *
+ * The velocity field \f$v\f$ is hard-coded in the current implementation since
+ * ScalarAdvection system is only used for testing limiters.
```

Comment:
You are adding a tag for the velocity field, so it's not entirely hard-coded I think? Do you mean it is not a dynamic field that would be evolved by some other system, but a static background field? In that case, perhaps write something like: `The velocity field \f$v\f$ is a background field for the purpose of this system, in the sense that it is not an evolved variable.`

Reply:
You are right. So the velocity field would be fixed now, but rather than explicitly writing numbers all around the places that require velocity field (and also adding comments such as "this number 1.0 is i-component of the hard-coded velocity field" etc), I thought it'd be better to make it as a tag and fix it through only one action if possible. I'll change words as you suggested :)